src/components/createNewBook/CreateNewBook.js

```diff
@@ -0,0 +1,9 @@
+const CreateNewBook = () => (
+  <form>
+    <input label="Title" type="text" placeholder="Book title" />
+    <input label="Author" type="text" placeholder="Book author" />
+    <input label="Create" type="submit" name="Create" value="Create" />
```

Comment:
- [ ] OPTIONAL - The input field doesn't have a label attribute. You can refer to the [react documentation](https://reactjs.org/docs/forms.html#:~:text=%3Cform%3E%0A%20%20%3Clabel%3E%0A%20%20%20%20Name%3A%0A%20%20%20%20%3Cinput%20type%3D%22text%22%20name%3D%22name%22%20/%3E%0A%20%20%3C/label%3E%0A%20%20%3Cinput%20type%3D%22submit%22%20value%3D%22Submit%22%20/%3E%0A%3C/form%3E) here on the proper way to assign a label to input fields.